cloud/security/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Security
+excerpt: Learn how your Timescale Cloud instance is secured
+product: cloud
+keywords: [security]
+---
+
+# Timescale Cloud security
+
+Learn how your Timescale Cloud instance is secured to protect your data and
+privacy.
+
+*   Get an [overview of Timescale Cloud security][overview]
+*   Learn how to [connect with a stricter SSL mode][ssl] for enhanced protection
+
```

Comment:
Feels like this is the area we should add content like MST has in https://docs.timescale.com/mst/latest/security/

Reply:
Ah ok! I didn't find it.